### PR TITLE
adds settings reference for metrics

### LINKED ISSUE #232: Document "Metrics Settings"
**Path to page with content (_do not_ include domain name):**
`/en/os/1.14.x/api/settings/`

**What content is incorrect?**
The settings reference lacks the "[Metrics Settings](https://github.com/bottlerocket-os/bottlerocket#metrics-settings)"

**What should the content say or communicate?**
There is an [ongoing effort](https://github.com/bottlerocket-os/bottlerocket-project-website/issues/203) to move the settings reference currently located on the [Bottlerocket README](https://github.com/bottlerocket-os/bottlerocket) to Bottlerocket.dev

This can be accomplished using the [Settings Documentation DSL](https://github.com/bottlerocket-os/bottlerocket-project-website/blob/main/METADOC-SETTING-DOCS-DSL.md). Effectively, the settings written out in the README's markdown needs to be converted to this DSL format. You can see an example of how this looks in the [ECS settings reference TOML file](https://github.com/bottlerocket-os/bottlerocket-project-website/blob/main/data/settings/1.14.x/ecs.toml) as well as [the actual ECS content markdown file](https://github.com/bottlerocket-os/bottlerocket-project-website/blob/main/content/en/os/1.14.x/api/settings/ecs/_index.markdown).

The file should be located at `/data/settings/1.14.x/metrics.toml` and `/content/en/os/1.14.x/api/settings/metrics/_index.markdown`.

### PR DESCRIPTION
**Pull request by stockholmux**
**Issue number:**

Closes #232 

**Description of changes:**
- Adds settings reference for `settings.metrics.*`

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

File: data/settings/1.15.x/metrics.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+If `false`, Bottlerocket will not send metrics.
+"""
+accepted_values = [
+    "`true`",
+    "`false`"
+]
+default = "`true` (except on `dev` variants, where it is `false`)"
+
+[[docs.ref.service-checks]]
+description = "A list of systemd services that Bottlerocket will checked to determine if a host is healthy."
```
Comment (arnaldo2792) [inline data/settings/1.15.x/metrics.toml:18]:
```suggestion
description = "A list of systemd services that Bottlerocket will check to determine if a host is healthy."
```

File: data/settings/1.15.x/metrics.toml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+default = """The default services vary by variant:
+
+- `apiserver` (all variants)
+- `chronyd` (all variants)
+- `containerd` (all variants)
+- `host-containerd` (all variants)
+- `docker` (`aws-ecs-*`, `aws-dev-*`)
+- `ecs` (`aws-ecs-*`)
+- `kubelet` (`*-k8s-*`)
```
Comment (arnaldo2792) [inline data/settings/1.15.x/metrics.toml:27]: I didn't know this. I added a new service that is required in k8s NVIDIA variants to be healthy. I'll check if we should add it to this list of services.

File: data/settings/1.15.x/metrics.toml (hunk 1 above)
Comment (stockholmux) [reply data/settings/1.15.x/metrics.toml]: good catch!

File: data/settings/1.15.x/metrics.toml (hunk 2 above)
Comment (stockholmux) [reply data/settings/1.15.x/metrics.toml:27]: Yeah, I pulled this from the `sources/models/shared-defaults/*.toml` files (under `service-checks`) when searching for the defaults.

**Review by arnaldo2792 — APPROVED** [review]

**Pull request merged**